packages/styled-components/src/models/StyledComponent.js

```diff
@@ -74,7 +72,7 @@ class StyledComponent extends Component<*> {
           // eslint-disable-next-line no-console
           console.warn(
             `Functions as object-form attrs({}) keys are now deprecated and will be removed in a future version of styled-components. Switch to the new attrs(props => ({})) syntax instead for easier and more powerful composition. The attrs key in question is "${key}" on component "${displayName}".`,
-            `\n ${(new Error()).stack}`
```

Comment:
Is there a reason why it was this way, maybe browser compat? Is this safe to change?

Reply:
idk, prettier removed it so I assume a safe transform